setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='ably',
-    version='0.8.1',
+    version='0.9.0',
     classifiers=[
         'Development Status :: 4 - Beta',
```

Comment:
Sure, looks like that should have been removed on `master` though a long time ago...